src/db/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as pgPromise from 'pg-promise'
+
+// TODO: Add ProcessUtils
+const ProcessUtils = {
+  ENV: {
+    debug: true,
+    pgSsl: true,
+    dbUrl: '',
+    pgUser: '',
+    pgDatabase: '',
+    pgPassword: '',
+    pgHost: '',
+    pgPort: 0,
+  },
+}
+
+// TODO: Add logging
+const logger = {
+  debug: console.debug,
+}
+
+const debugOptions = {
+  query: (e: pgPromise.IEventContext) => {
+    logger.debug(`QUERY: ${e.query}`)
+    if (e.params) {
+      logger.debug(`PARAMS: ${JSON.stringify(e.params)}`)
+    }
+  },
+}
+
+const initOptions = {
+  ...(ProcessUtils.ENV.debug ? debugOptions : {}),
+}
+
+const pgp = pgPromise(initOptions)
+
+// Timestamp will automatically be converted to UTC time-zone - No need to convert in select queries anymore
+// 1114 is OID for timestamp in Postgres
+pgp.pg.types.setTypeParser(1114, (str) => new Date(`${str} GMT`))
```

Comment:
PG_OID_TIMESTAMP = 1114
?

Reply:
Thank you for your comment, as discussed separately:
Will keep as is for now, because the value is used only here